boards/DCM/Src/App/App_DcmWorld.c

```diff
@@ -111,3 +113,8 @@ struct Clock *App_DcmWorld_GetClock(const struct DcmWorld *const world)
 {
     return world->clock;
 }
+
+struct Imu *App_DcmWorld_GetImu(const struct DcmWorld *const world)
+{
+    return world->imu;
+}
```

Comment:
u gotta reread ur code. make sure this is under the buzzer

Reply:
ok. I looked around and this was the last remaining place where they were out of line